checker_test.go

```diff
@@ -2330,10 +2330,23 @@ error:
 error:
   unexpected success
 got:
-  []uint8{0x6e, 0x75, 0x6c, 0x6c}
+  []uint8("null")
 want:
   nil
 `,
+}, {
+	about:   "JSONEquals with RawMessage",
+	checker: qt.JSONEquals,
+	got:     []byte("null"),
+	args:    []interface{}{json.RawMessage("null")},
+	expectedNegateFailure: `
+error:
+  unexpected success
+got:
+  []uint8("null")
+want:
+  json.RawMessage("null")
```

Comment:
Very nice!

Reply:
nice, isn't it? I only realised that you could do that this morning, when someone was asking for the equivalent of https://godoc.org/github.com/stretchr/testify/assert#JSONEq, which tests against a string.